supertokens_python/recipe/session/utils.py

```diff
@@ -429,7 +451,7 @@ def validate_and_normalise_user_input(
         )
 
     async def anti_csrf_func(req: BaseRequest, user_context: Any) -> str:
-        cookie_same_site_val = cookie_same_site
+        cookie_same_site_val = await cookie_same_site_func(req, user_context)
```

Comment:
```suggestion
        cookie_same_site = await cookie_same_site_func(req, user_context)
```

Maybe just call it `cookie_same_site` now